MANIFEST.in

```diff
@@ -1,8 +1,7 @@
 include ChangeLog
-include COPYING
```

Comment:
I am fine if it is also listed in setup.cfg -- for me that's actually separate information, i.e. "this is a file containing the license" while MANIFEST.in is "please include this file in the source distribution".

Reply:
It's a file specifically called out in the metadata, just like `py_modules` or `packages`. But instead of declaring it as an importable source file, it's declared as a dist-info file which must be installed as wheel metadata.

I disagree with your analysis -- I don't view this as guessable logic -- but if you still want it I'll put it back.

Reply:
Ok, will do.

Reply:
Thank you! Apologies for being a pain about it. :|

Reply:
lol, I wish every PR review was this simple to solve.

"Looks good, just please include this minor nitpicky line that I really like even though it's a no-op."

(I mean, it's lots better than "I disagree with the functionality and would rather you write your own separate program to do X". I've gotten that once or twice for stuff I thought was a no-brainer bugfix.)